src/CompanyService/Startup.cs

```diff
@@ -179,10 +179,9 @@ private void ConfigureValidators(IServiceCollection services)
 
         private void ConfigureMappers(IServiceCollection services)
         {
-            services.AddTransient<IMapper<DbPosition, PositionResponse>, PositionMapper>();
-            services.AddTransient<IMapper<Position, DbPosition>, PositionMapper>();
-
-            services.AddTransient<IMapper<NewDepartmentRequest, DbDepartment>, DepartmentMapper>();
+            services.AddTransient<IDbPositionMapper, Mappers.RequestMappers.DbPositionMapper>();
+            services.AddTransient<IDepartmentMapper, Mappers.ResponsesMappers.DepartmentMapper>();
+            services.AddTransient<IDbDepartmentMapper, global::CompanyService.Mappers.RequestMappers.DbDepartmentMapper>();
```

Comment:
What is ‘global’ key-word means? And why is it here?

Reply:
fixed